addons/sms/models/mail_thread.py

```diff
@@ -229,8 +230,8 @@ def _message_sms(self, body, subtype_id=False, partner_ids=False, number_field=F
             subtype_id = self.env['ir.model.data']._xmlid_to_res_id('mail.mt_note')
 
         return self.message_post(
-            body=plaintext2html(html2plaintext(body)), partner_ids=partner_ids or [],  # TDE FIXME: temp fix otherwise crash mail_thread.py
-            message_type='sms', subtype_id=subtype_id,
+            body=render_sms_notification(body), partner_ids=partner_ids or [],  # TDE FIXME: temp fix otherwise crash mail_thread.py
```

Comment:
Would be worth a small explanation of what we expect as body in parameter docstring, now that the flow is more asserted.